index.js

```diff
@@ -0,0 +1,167 @@
+// FEATURED SECTION
+
+const container = document.querySelector('#featured');
+
+const features = [
+  {
+    thumbnail: 'images/first.jpg',
+    name: 'Osandi Robinson',
+    occupation: 'Designer in Charlotte, NC, United States',
+    description:
+      'Mentored at Apple, Osandi has spent over ten years helping teams.',
+  },
+  {
+    thumbnail: 'images/second.jpg',
+    name: 'Danielle Thompson',
+    occupation: 'Designer in Singapore, Singapore',
+    description:
+      'Danielle is a seasoned UI/UX product designer specializing in the sports and fintech market.',
+  },
+  {
+    thumbnail: 'images/third.jpg',
+    name: 'Mike Slone',
+    occupation: 'Designer in Ladera Ranch, CA, United States',
+    description:
+      'Mike is a highly experienced and successful product, UX, UI, visual, and branding designer.',
+  },
+  {
+    thumbnail: 'images/fourth.jpg',
+    name: 'Amélie Lamont',
+    occupation: 'Designer in New York, NY, United States',
+    description:
+      'Amélie is an independent product designer, writer, and UX teacher based in New York.',
+  },
+  {
+    thumbnail: 'images/fivth.jpg',
+    name: 'Lee Hsieh',
+    occupation: 'UX/UI Designer in Tokyo, Japan',
+    description:
+      'Lee has designed everything from sex education software for kids to flight management apps for pilots.',
+  },
+  {
+    thumbnail: 'images/sixth.jpg',
+    name: 'Greg Kujda',
+    occupation: 'Designer in Long Beach, CA, United States',
+    description:
+      'Greg is a hybrid storyteller with deep experience leading UI/UX, visual, and motion design.',
+  },
+];
+
+// CREATING FEATURED SECTION FROM ARRAY DATA AND POPULATING DYNAMICALLY
+
+const populate = function (number) {
+  function generate(items) {
+    items.forEach((feature) => {
+      const content = ` <div class="featured-items">
```

Comment:
- [ ] (Optional) It would be nice to change the `div` tag to an `article` tag for semantic purposes 👍